src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -103,50 +103,14 @@ protected bool ShouldProcessServiceOperation(ServiceController service)
             string message = StringUtil.Format(errorMessage,
                 serviceName,
                 displayName,
-                (null == innerException) ? "" : innerException.Message
-                );
-            ServiceCommandException exception =
-                new ServiceCommandException(message, innerException);
-            exception.ServiceName = serviceName;
+                (null == innerException) ? "" : innerException.Message);
```

Comment:
Can we use `String.Empty`?

Reply:
Will do.

Reply:
Please clarify for me: I found many "*" strings in cs files - should we create a constant for it?